apps/hash-frontend/src/pages/[shortname]/types/entity-type/[entity-type-id].page.tsx

```diff
@@ -58,13 +58,21 @@ const Page: NextPageWithLayout = () => {
 
   const draftEntityType = useMemo(() => {
     if (router.query.draft) {
-      // @todo use validation when validateEntityType doesn't return undefined
```

Comment:
Not entirely sure why we opted to just not use it, probably an oversight

Reply:
This would have informed us that our draft entity type was broken
![image](https://user-images.githubusercontent.com/25749103/220990526-9dfdc0e7-5ac0-4ed8-984c-91c56b90ceff.png)